sdk/avalon_sdk/work_order_receipt/work_order_receipt.py

```diff
@@ -143,7 +143,7 @@ def update_receipt(self, work_order_id, update_type,
             wo_resp_hash = crypto_utility.compute_message_hash(wo_resp_bytes)
             wo_resp_hash_str = crypto_utility.byte_array_to_hex(wo_resp_hash)
             data = wo_resp_hash_str
-        public_key = signing_key.verifying_key.to_pem()
+        public_key = signing_key.verifying_key.to_pem().decode("ascii")
```

Comment:
Please use the `crypto_utility` function `crypto_utility.get_verifying_key(signing_key)`

https://github.com/hyperledger/avalon/blob/master/common/crypto_utils/avalon_crypto_utils/crypto_utility.py#L44

Reply:
Also needs to be corrected in https://github.com/hyperledger/avalon/blob/master/enclave_manager/avalon_enclave_manager/work_order_kv_delegate.py#L38